xds/bootstrap/bootstrap_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package bootstrap
+
+import (
+	"encoding/json"
+	"testing"
+
+	"google.golang.org/grpc/credentials"
+)
+
+const testCredsBuilderName = "test_creds"
+
+var builder *testCredsBuilder
+
+func init() {
+	// Register a new credential builder.
+	builder = &testCredsBuilder{}
```

Comment:
Nit: move this to the global variable declaraion.

```go
var builder = &testCredsBuilder{}
```

Reply:
Done